hedera-node/src/main/resources/feeSchedules.json

```diff
@@ -837,12 +840,12 @@
           "min" : 0,
           "max" : 1000000000000000
         }
-      }
+      }]
     }
   }, {
     "transactionFeeSchedule" : {
       "hederaFunctionality" : "TokenMint",
```

Comment:
Shouldn't we have 2 rows here?
Same thing is applied for: `TokenBurn` & `TokenWipe`